cirq-core/cirq/ops/measure_util_test.py

```diff
@@ -60,3 +60,39 @@ def test_measure_each():
         cirq.measure(a, key='a!'),
         cirq.measure(b, key='b!'),
     ]
+
+
+def test_measure_single_paulistring():
+    # Correct application
+    q = cirq.LineQubit.range(3)
+    ps = cirq.X(q[0]) * cirq.Y(q[1]) * cirq.Z(q[2])
+    assert cirq.measure_single_paulistring(ps, key='a') == cirq.PauliMeasurementGate(
+        ps.values(), key='a'
+    ).on(*ps.keys())
+
+    # Empty application
+    with pytest.raises(ValueError, match='should be an instance of cirq.PauliString'):
+        _ = cirq.measure_single_paulistring(cirq.I(q[0]) * cirq.I(q[1]))
+
+    # Wrong type
+    with pytest.raises(ValueError, match='should be an instance of cirq.PauliString'):
+        _ = cirq.measure_single_paulistring(q)
```

Comment:
Would this be worth splitting into multiple separate tests ?

Reply:
I've followed the pattern of `test_measure_qubits` and `test_measure_each`, so I guess it's fine to have consistency in the file.